server/app/views/applicant/ProgramCardsSectionParamsFactory.java

```diff
@@ -129,8 +129,8 @@ public ProgramCardParams getCard(
     if (detailsUrl.isEmpty() || detailsUrl.isBlank()) {
       detailsUrl =
           profile.isPresent() && applicantId.isPresent()
-              ? applicantRoutes.show(profile.get(), applicantId.get(), program.id()).url()
-              : applicantRoutes.show(program.id()).url();
+              ? applicantRoutes.review(profile.get(), applicantId.get(), program.id()).url()
```

Comment:
**question (non-blocking):** Will this also be replaced with the new program details page once it's ready?  If so, it could be handy to add a TODO here like we have in the `NorthStarApplicantIneligibleView`.